src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -77,12 +77,36 @@
     {:population (ds/select-columns (ds/join-rows population update-births)
                                     [:gss-code :sex :age :year :popn])}))
 
+(defworkflowfn remove-deaths
+  "Takes in a dataset of aged on popn estimates with births added, and
+   a dataset of births by sex for the current projection year (last year
+   of population dataset).
+   Returns a dataset where the deaths output from the mortality module is
+   subtracted from the latest year in the popn dataset."
+  {:witan/name :ccm-core/remove-deaths
+   :witan/version "1.0"
+   :witan/input-schema {:population PopulationSchema :deaths DeathsSchema}
+   :witan/param-schema {:_ nil}
+   :witan/output-schema {:population PopulationSchema}}
+  [{:keys [population deaths]} _]
+  (let [latest-yr (get-last-yr-from-popn population)
```

Comment:
Every function is performing this operation. Worth pulling it out into the args i reckon.